lib/components/common/PreviewTable.tsx

```diff
@@ -1,123 +1,123 @@
-import { Box, Table, TableBody, TableCell, TableHead, TableRow } from "@mui/material"
```

Comment:
I ran the lint command and a couple of files got changes only related with linting. This is one of them